app/Http/Controllers/PostController.php

```diff
@@ -28,7 +29,7 @@ public function index(Request $request)
                     $queryBuilder->whereHas('user', function ($q) use ($hashtag) {
                         $q->whereRaw('BINARY name LIKE ?', ['%'.$hashtag.'%']);
```

Comment:
Fajnie, że znalazłeś whereRaw, ale możesz to zrobić where('name', 'ilike', '%$hashtag%');